src/plugins/data/public/ui/typeahead/suggestions_component.tsx

```diff
@@ -88,14 +88,14 @@ export default class SuggestionsComponent extends PureComponent<SuggestionsCompo
         inputContainer={this.props.inputContainer}
         suggestionsSize={this.props.size}
       >
-        {(containerWidth: number) => (
+        {(rect: DOMRect) => (
```

Comment:
This is the culprit: children prop received `rect`, but handled as `containerWidth`, so isDescriptionFittable logic broke and always resolved to `false`
Improved types of a wrapper component to catch this